app/controllers/challenge_assignments_controller.rb

```diff
@@ -219,9 +219,10 @@ def default_all
   def default
     @challenge_assignment.defaulted_at = Time.now
     @challenge_assignment.save
-    @challenge_assignment.collection.notify_maintainers("Challenge default by #{@challenge_assignment.offer_byline}",
-        "Signed-up participant #{@challenge_assignment.offer_byline} has defaulted on their assignment for #{@challenge_assignment.request_byline}. " +
-        "You may want to assign a pinch hitter on the collection assignments page: #{collection_assignments_url(@challenge_assignment.collection)}")
+    subject = t("user_mailer.collection_notification.challenge_default.subject", offer_byline: @challenge_assignment.offer_byline )
+    message = t("user_mailer.collection_notification.challenge_default.complete", offer_byline: @challenge_assignment.offer_byline, request_byline: @challenge_assignment.request_byline, assignments_page_link: collection_assignments_url(@challenge_assignment.collection))
+    @challenge_assignment.collection.notify_maintainers(subject, message)
```

Comment:
The `t()` calls here should also be pulled into `I18n.with_locale` block in the `notify_maintainers` method, which could maybe be renamed to `notify_maintainers.challenge_default` to signify that specialization/split of the method.